mint/build/admin/subnet-health-usage.go

```diff
@@ -0,0 +1,57 @@
+/*
+*
+*  Mint, (C) 2021 Minio, Inc.
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+ */
+
+package main
+
+import (
+	"context"
+	"time"
+)
+
+// testSubnetHealthUsage tests that the usage data (bucket count, object count and usage)
+// is present (and non-zero) in the output of the healthinfo api. It is assumed that the
+// test always runs against a cluster with non-zero usage.
+func testSubnetHealthUsage() {
+	function := "testSubnetHealthUsage"
+	startTime := time.Now()
+	args := map[string]interface{}{}
+
+	si, err := adminClient.ServerInfo(context.Background())
```

Comment:
Are you planning to run add subnet health or just this?? or `adminClient.ServerHealthInfo()` ?

Reply:
In this PR I thought of adding the test only for the regression bug that was discovered recently, and that requires only `adminClient.ServerInfo` call. More tests can be added subsequently in separate PRs I think.

I'm open to adding more tests in this PR itself (including `ServerHealthInfo` call) if you feel that's important.